### ISSUE TITLE
Uncollectible and heroes support.

### ISSUE THREAD

**Issue by pit00** [mtimkovich/hearthsounds#3]
Hi!
Any way to support the sound of uncollectible cards and Heroes (the new DK heroes)?
I think this change will work:

```
r = requests.get('http://www.hearthpwn.com/cards/minion',
                     params={'filter-name': query, 'filter-premium': 1})
```
to

```
r = requests.get('http://www.hearthpwn.com/cards',
                     params={'filter-name': query, })
```
Or put a flag.

**Comment by mtimkovich** [comment]
Thanks for the suggestion! I added this in commit 1ba5ce2f121a87d0dfb8015c38efb89fb329689e, and should be live on the website now.

**Comment by pit00** [comment]
Amazing!

**Comment by pit00** [comment]
Just one thing, the uncollectible cards are not available
Like [Defender](http://www.hearthpwn.com/cards/318-defender) and [Imp](http://www.hearthpwn.com/cards/14642-imp).

**Comment by mtimkovich** [comment]
I'll add a checkbox for it later. I don't want it on by default, otherwise it'll return useless cards like this: http://www.hearthpwn.com/cards/14711-golemagg

**Comment by pit00** [comment]
Agreed!

**Comment by mtimkovich** [comment]
This is implemented now! fb01244e2fcc5550930431ad7d2628b66ae30a59 Thanks for your suggestion!

**Comment by pit00** [comment]
Sweet!
